### ISSUE TITLE
plumed.org updates

### ISSUE THREAD

**Issue by carlocamilloni** [plumed/plumed.github.io#7]
@maxbonomi @GiovanniBussi @gtribello 
I think we need to clean and update the plumed.org website also to reduce the maintance efforts.
I have just added a direct link to the Tutorial page and moved around some button. 

This is what I think may work:
- [ ] In general the PLUMED/NEST button could be direct link also in the NEST and in the Tutorial pages
- [x] We could remove the NEWS page that is outdated
- [ ] We need to update the FUNDING to ackwnoledge the recent CECAM and most importantly PNRR
- [x] Shall we remove the Forum page and have a sentence in the Manuals (former DOC) page?
- [ ] Add Daniele in the PEOPLE page
- [ ] Other ideas?

**Comment by GiovanniBussi** [comment]
I agree with everything. For consistency I would suggest that we either keep the separate nest page or we put the link to nest in the text (and not as a tab). And viceversa for the link to the main site from the nest. But I am not 100% convinced, I am ok with all solutions

I was also wondering if we can move the "push" site for plumed school directly to the plumed organization so that it will then look like: plumed.org/school

**Comment by gtribello** [comment]
I agree also.  What repo do you have to edit to edit the plumed.org page?

Was wondering for the masterclass page.  What if we add something to the school page so that you can search the school browse page and get just the masterclasses up.  We can then replace the link to the masterclasses with a a link to school browse that puts "masterclass" in the search box.

I think this would delete a few of the masterclasses that appear on the tutorial page.  The first one that would disappear is the one that is just Giovanni and Max telling people what the second masterclass is.  The others are ones where I have emailed the authors of the class all the stuff they need to transfer it to the school page and they haven't done it.  I should point through I went through and converted all their stuff from Doxygen to Markdown for them.  All they had to do was spend five minutes putting it a GitHub repo and emailing me the archive address.

**Comment by carlocamilloni** [comment]
I like the idea of browsing masterclass as a subpart of tutorials, I think it will avoid usell duplication of the site to search for something.

The repository of plumed.org is the one of this issue https://github.com/plumed/plumed.github.io

**Comment by carlocamilloni** [comment]
> I was also wondering if we can move the "push" site for plumed school directly to the plumed organization so that it will then look like: plumed.org/school

I agree
